doc/architecture/decisions/0002-store-subset-of-syndication-data.md

```diff
@@ -0,0 +1,28 @@
+# 2. Store subset of Syndication data
```

Comment:
I don't think this record needs to be explicit about only a subset of the data being stored. IMO, the decision is about creating a local cache of the data with the reasons for doing so being:
* to prevent querying syndication in realtime
* insulating the applications from a failure of syndication
* provides the ability to transform the data into a more useful format

I don't know if it would be better having the decision to transform the data as a separate record?